cmd/ks-apiserver/app/server.go

```diff
@@ -176,6 +176,7 @@ func waitForResourceSync() {
 	informerFactory.Batch().V1().Jobs().Lister()
 	informerFactory.Batch().V1beta1().CronJobs().Lister()
 	informerFactory.Extensions().V1beta1().Ingresses().Lister()
+	informerFactory.Autoscaling().V2beta2().HorizontalPodAutoscalers().Lister()
```

Comment:
why do we call Lister() ? I do think it's necessary

Reply:
If the lister is not initialized before informerFactory.Start(), the data will not be cached.

Reply:
this is not the right way to do it, I'll change this later